schema.json

```diff
@@ -1378,7 +1377,7 @@
     "Certificate": {
       "description": "",
       "type": "object",
-      "required": ["CertificateLanguages", "CommercialTransaction", "ProductDescription", "Inspection", "Validation"],
+      "required": ["CertificateLanguages", "CommercialTransaction", "ProductDescription", "Inspections", "Validation"],
```

Comment:
Could we imagine a softer breaking change by allowing either Inspection or Inspections (using oneOf keyword)?
We could maintain compatibility with the previous version and that would truly help the update of the schema-tools

Reply:
Request applied.